portlets/src/main/webapp/vue-app/realizations/components/Realizations.vue

```diff
@@ -189,6 +194,12 @@ export default {
       }
     },
   },
+  mounted() {
```

Comment:
this fix, has resolved the dynamic header , 
you can find bellow a screencasted issue 
![Screenshot from 2022-09-02 18-20-05](https://user-images.githubusercontent.com/101107377/188207000-04101a54-a2a5-4153-a240-a5f6e7048409.png)
@boubaker

Reply:
nope, the problem is that status column is missing, I will add it to fix this

Reply:
can you test again on your side as well @ArfaouiMohamedAmine ? If it's ok, no further comment for my part. Thanks.

Reply:
okay am on it !

Reply:
@boubaker  it works just fine ! thank you !

Reply:
You 're welcome :-)